mailpoet/views/newsletter/templates/blocks/coupon/settings.hbs

```diff
@@ -54,7 +54,53 @@
     </summary>
     <div className="content">
 
-      Content here
+      <div class="mailpoet_form_field">
+        <label>
+          <div class="mailpoet_form_field_title mailpoet_form_field_title_inline"><%= __('Minimum spend') %></div>
+          <div class="mailpoet_form_field_input_option">
+            <input type="text" name="minimum-amount" class="mailpoet_input mailpoet_field_coupon_minimum_amount" value="{{ model.minimumAmount }}" placeholder="<%= __('No minimum')%>" />
```

Comment:
I'm not super familiar with the technical stack of the email editor, so I'm unsure about how sanitizing input and escaping output works here. Do we need to do something about it or is one of the libraries that we are using already taking care of both?

Reply:
> Do we need to do something about it or is one of the libraries that we are using already taking care of both?

Yes, we have already taken care of this.

But a few improvements can be made, and I will implement that in the next commit.

Reply:
Thanks, could you please detail what we are already doing to take care of this and what few improvements still needed to be made that you implemented? This information will help when reviewing the code.

Also, I wonder if there is a reason to combine the changes to improve escaping and sanitizing with the changes to support different decimal separators, and I believe indentation changes as well, in a single commit (cabdbcd)? From my perspective, this makes reviewing this PR a bit harder and it might make understanding the changes proposed here slightly more difficult if in the future we need to debug this part of the code. I would suggest splitting those changes into multiple commits. What do you think?

Reply:
> could you please detail what we are already doing to take care of this

We perform the sanitizing (and user input escape) when the Newsletter is saved [here](https://github.com/mailpoet/mailpoet/blob/trunk/mailpoet/lib/Newsletter/NewsletterSaveController.php#L116).

We are using Handlebarsjs for our templates and escaping the output with `{{}}`. [read more here](https://handlebarsjs.com/guide/expressions.html#html-escaping)

> what few improvements still needed to be made that you implemented?

There's generally no need to add an escape here as it serves for Html placeholder not user input.
I thought to add an [improvement](https://github.com/mailpoet/mailpoet/blob/cabdbcd4b9b495a24874303b2cd059ca32a173b7/mailpoet/views/newsletter/templates/blocks/coupon/settings.hbs#L113) there to be safe.

Reply:
> Also, I wonder if there is a reason to combine the changes to improve escaping and sanitizing with the changes to support different decimal separators, and I believe indentation changes as well, in a single commit (https://github.com/mailpoet/mailpoet/commit/cabdbcd4b9b495a24874303b2cd059ca32a173b7)? From my perspective, this makes reviewing this PR a bit harder and it might make understanding the changes proposed here slightly more difficult if in the future we need to debug this part of the code. I would suggest splitting those changes into multiple commits. What do you think?

Done